src/comm/Connection.cc

```diff
@@ -180,6 +178,18 @@ Comm::Connection::connectTimeout(const time_t fwdStart) const
     return min(ctimeout, ftimeout);
 }
 
+void
+Comm::Connection::setIdent(const Ident::User &anIdent)
+{
+    static const SBuf nil("nil");
+    if (ident) {
+       debugs(5, 3, "ignore rewriting " << ident.value() << " with " << (anIdent ? anIdent.value() : nil));
+       return;
+    }
+    debugs(5, 3, "got ident: " << (anIdent ? anIdent.value() : nil));
```

Comment:
Please use value_or() to simplify this debugs() and the one above it.

Reply:
Done.